addons/website_sale_product_configurator/controllers/main.py

```diff
@@ -44,8 +44,15 @@ def _prepare_product_values(self, product, category, search, **kwargs):
         values['optional_product_ids'] = [p.with_context(active_id=p.id) for p in product.optional_product_ids]
         return values
 
-    @http.route(['/shop/cart/update_option'], type='http', auth="public", methods=['POST'], website=True, multilang=False)
-    def cart_options_update_json(self, product_and_options, goto_shop=None, lang=None, **kwargs):
+    @http.route(
+        '/shop/cart/update_option',
+        type='json',
+        auth='public',
+        methods='POST',
```

Comment:
I don't see any occurrence of this in Odoo :thinking: (and runbot does not seem to like it)
```suggestion
        methods=['POST'],
```

Reply:
My bad, I though the brackets were not needed :sweat_smile: